docs/.vitepress/theme/coffee-machine.css

```
@@ -0,0 +1,165 @@
+
+.coffee-hero h1,
+.coffee-page-header h1 {
+  max-width: 780px;
+  margin: 8px 0 16px;
+  font-size: clamp(42px, 7vw, 76px);
+  line-height: .98;
+  letter-spacing: -.045em;
+}
+
+.coffee-page-header h1 { font-size: clamp(38px, 6vw, 64px); }
+.coffee-hero > p:not(.coffee-eyebrow), .coffee-page-header > p:not(.coffee-eyebrow) { max-width: 700px; font-size: 1.16rem; color: var(--coffee-muted); }
```

Stylelint (17.14.0)

[error] Too many declarations, maximum 1 (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)